src/pages/index.astro

```diff
@@ -214,6 +219,10 @@ const t = useTranslations(lang)
 </Base>
 
 <style>
+  html {
+    scroll-behavior: smooth;
```

Comment:
Let's ensure [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) is respected.